src/app/store.ts

```diff
@@ -1,10 +1,16 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { todosSlice } from '../features/todos';
+import { filterSlice } from '../features/filter';
+import { currentTodoSlice } from '../features/currentTodo';
+import { TypedUseSelectorHook, useSelector } from 'react-redux';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(todosSlice, filterSlice, currentTodoSlice);
 
 export const store = configureStore({
   reducer: rootReducer,
 });
 
+export const selector: TypedUseSelectorHook<RootState> = useSelector
```

Comment:
The export `selector` should be renamed to `useTypedSelector` to better reflect its purpose as a typed version of `useSelector`. This will improve code readability and maintainability.